core/core.go

```diff
@@ -524,6 +525,8 @@ func NewTextile(ctx context.Context, conf Config, opts ...Option) (*Textile, err
 
 	// Start pulling threads
 	t.tn.StartPulling()
+	// Start republishing ipns keys
+	bs.StartRepublishing("*/5 * * * *", t.internalHubSession)
```

Comment:
will make this a flag so we can tune with the expiration settings

Reply:
You'll have to stop it too in the core `Close` method. Something like,
```
ctx := cron.Stop()
<-ctx.Done()
log.Info("republisher was shutdown")
```

If you created a `Close` method on `Service` and put something like the above in there, that could be called externally in the core `Close` method.